database/DataGenerator/bedSize.js

```diff
@@ -0,0 +1,11 @@
+module.exports = [
+  {
```

Comment:
@AndrewTjandra For this section, I think it could make more sense to just store these options as an array (unless you plan to add more keys over time). That way you don't have to have 3 separate objects with a redundant size property.